flowmachine/tests/test_spatial_activity.py

```diff
@@ -1,152 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-"""
-Tests for the spatial activity class
-"""
-
-# since the directory is not a module, it cannot be imported
-# we use this temporary hack for sourcing utils.py while the testing framework is not completely refactored
-import os
-
-exec(open(os.path.join(os.path.dirname(__file__), "utils.py")).read())
-from unittest import TestCase
-from pandas import DataFrame
-
-from flowmachine.features import TotalLocationEvents
-
-
-class test_total_events_location(TestCase):
-    def test_events_at_cell_level(self):
-        """
-        TotalLocationEvents() returns data at the level of the cell.
-        """
-
-        te = TotalLocationEvents("2016-01-01", "2016-01-04", level="versioned-site")
-        df = te.get_dataframe()
-        self.assertIs(type(df), DataFrame)
-
-        # Test one of the values
-        df.date = df.date.astype(str)
-        val = list(
-            df[
-                (df.date == "2016-01-03") & (df.site_id == "zArRjg") & (df.hour == 17)
-            ].total
-        )[0]
-        self.assertEqual(val, 3)
-
-    def test_ignore_texts(self):
-        """
-        TotalLocationEvents() can get the total activity at cell level excluding texts.
-        """
-        te = TotalLocationEvents(
-            "2016-01-01", "2016-01-04", level="versioned-site", table="events.calls"
-        )
-        df = te.get_dataframe()
-        self.assertIs(type(df), DataFrame)
-        # Test one of the values
-        df.date = df.date.astype(str)
-        val = list(
-            df[
-                (df.date == "2016-01-01") & (df.site_id == "0xqNDj") & (df.hour == 3)
-            ].total
-        )[0]
-        self.assertEqual(val, 3)
-
-    def test_only_incoming(self):
-        """
-        TotalLocationEvents() can get activity, ignoring outgoing calls.
-        """
-        te = TotalLocationEvents(
-            "2016-01-01", "2016-01-04", level="versioned-site", direction="in"
-        )
-        df = te.get_dataframe()
-        self.assertIs(type(df), DataFrame)
-        # Test one of the values
-        df.date = df.date.astype(str)
-        val = list(
-            df[
-                (df.date == "2016-01-01") & (df.site_id == "6qpN0p") & (df.hour == 0)
-            ].total
-        )[0]
-        self.assertEqual(val, 2)
-
-    def test_events_daily(self):
-        """
-        TotalLocationEvents() can get activity on a daily level.
-        """
-        te = TotalLocationEvents(
-            "2016-01-01", "2016-01-04", level="versioned-site", interval="day"
-        )
-        df = te.get_dataframe()
-        self.assertIs(type(df), DataFrame)
-        self.assertEqual(
```

Comment:
I presume this columns check was removed in the converted version because it already exists elsewhere?

Reply:
Covered here: https://github.com/Flowminder/FlowKit/pull/94/files#diff-152b8e433482382027342c580027bd09R16